src/test/resources/Latencies.html

```diff
@@ -0,0 +1,184 @@
+<xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Yes! But not always! What I mean is that the html files contains all the use cases but once could modify the simulator to limit the options, but still be able to reuse the same report file.
Also, there's no groovy here. In groovy, I would have generated the html file. But in java ? Heck... no... This would require an external library and eventually a template... So sad :-(

Reply:
Although there is: https://j2html.com/examples.html